### PR TITLE
Update callaudiod to version 0.1.3

### PR DESCRIPTION
**Pull request by marek-lach**

### CODE REVIEW

File: PKGBUILDS/phosh/callaudiod/PKGBUILD [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+pkgver=0.1.3
 pkgrel=1
 pkgdesc="Call audio routing daemon"
 url="https://gitlab.com/mobian1/callaudiod"
 license=('GPL3')
 arch=('x86_64' 'armv7h' 'aarch64')
-depends=('alsa-lib' 'glib2' 'libpulse')
-makedepends=('meson')
+depends=('alsa-lib' 'glib2' 'libpulse' 'pulseaudio')
+makedepends=('meson' 'git')
```
Comment (Danct12) [inline PKGBUILDS/phosh/callaudiod/PKGBUILD:10]: This is a tarball, so we don't need `git` in makedepends.

**Comment by Danct12** [comment]
Overall, LGTM but the commit name doesn't follow [the guidelines](https://github.com/dreemurrs-embedded/Pine64-Arch/blob/master/CONTRIBUTING.md#commit-message-guidelines).

**Comment by Danct12** [comment]
Please squash the commits.

**Comment by marek-lach** [comment]
@Danct12 I believe this is now ready to be merged.

**Comment by Danct12** [comment]
Your commit name still doesn't follow the guidelines, change it to `pkg: phosh: callaudiod: upgrade to 0.1.3`

**Pull request closed without merging**